part1/unicafe/src/App.jsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from 'react'
+
+// a proper place to define a component
+const Statistics = ({presentStatistics, good, neutral, bad}) => {
+  // ...
+
+  const total = {good} + {neutral} + {bad}
+
+  if (presentStatistics === 0) {
+    return (
+      <p>No feedback given</p>
+    )
+  }
+  return (
+    <div>
+      <tr>
+        <td><StatisticLine text='good' /></td>
+        <td><StatisticLine value={good} /></td>
+      </tr>
```

Comment:
```suggestion
  }
  
  const TableElement = (leftCell, rightCell) => return (
      <tr>
        <td><StatisticLine text={leftCell} /></td>
        <td><StatisticLine value={rightCell} /></td>
      </tr>
  )
  
  return (
    <div>
      <TableElement leftCell='good' rightCell={good} />
      <TableElement leftCell='neutral' rightCell={neutral} />
      <TableElement leftCell='bad' rightCell={bad} />
```

A good opportunity to compact it into a reusable component